### PR TITLE
add bio for Ski Season

### LINKED ISSUE #14: Add a bio file for skiseason
- [x] create a branch
- [x] add bio file
- [x] open a pull request
- [x] get some feedback
- [x] merge the pull requeset

### PR DESCRIPTION
**Pull request by skiseason**
This pull request adds bio for ski seasoon

closed #14

### CODE REVIEW

File: skiseason.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Ski Season
+* Name: Bob
+* Location: USA
```
Comment (githubteacher) [inline skiseason.md:3]: What did you eat for breakfast this morning?

Comment (rwspiegel) [inline skiseason.md:3]: When does the snow start?

**Comment by skiseason** [comment]
I added season start date...  I can't predict natural snow, but man-made snow will arrive then.  Sound Good?

File: skiseason.md (hunk 1 above)
Comment (skiseason) [inline skiseason.md:3]: I added season start date... I can't predict natural snow, but man-made snow will arrive then. Sound Good?

Comment (githubteacher) [inline skiseason.md:3]: :+1: Looks good to me!

**Pull request merged**